sdb/stocksdashboard.py

```diff
@@ -195,7 +196,7 @@ def get_params(self, params, name, color):
 
     def plot_stock(self, input_data=None, p=None, column='adj_close',
                    title="Stock Closing Prices", ylabel='Price',
-                   add_hover=True, params={}, **kwargs):
+                   add_hover=True, params={}):
```

Comment:
I think we should be able to give additional params to the plots that are general to all the plots.